src/Forms/ShowHistory.cs

```diff
@@ -113,7 +113,7 @@ private void ButtonCompareClick(object sender, EventArgs e)
                 using (var compareForm = new Compare())
                 {
                     compareForm.Initialize(_subtitle, Configuration.Settings.Language.General.CurrentSubtitle, h2.Subtitle, descr2);
-                    compareForm.Show();
+                    compareForm.ShowDialog();

```

Comment:
After re-reading docu I experimented a bit. Makes quite a difference.
Thank you, for the tip.